CommunityFridgeMapApi/functions/dev/get_all/app.py

```diff
@@ -0,0 +1,70 @@
+import os
+import logging
+from botocore.exceptions import ClientError
+import json
+import sys
+try:
+    from db import get_ddb_connection
+except:
+    #If it gets here it's because we are performing a unit test. Here is an example of someone having a similar issue
+    #https://stackoverflow.com/questions/69592094/pytest-failing-in-aws-sam-project-due-to-modulenotfounderror
+    from dependencies.python.db import get_ddb_connection
```

Comment:
@abhinav1912 we are doing this for the unit tests. 

Testing functions with layers is a little weird, so need to do this workaround. 

When doing unit testing the layer isn't installed locally so it can't find the **db** package